### PR TITLE
[Remove Stream/Optional usage: portal-workflow-metrics-rest-impl]Lps 178425 3 23

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-178425

### CODE REVIEW

**Comment by slnn** [comment]
ci:test:workflow

**Comment by Kvale1733** [comment]
ci:test:workflow-metrics

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/dto/v1_0/util/ProcessUtil.java [hunk 1]
```diff
@@ -68,26 +66,28 @@ public static Process toProcess(Document document, Locale locale) {
...
-				entry -> {
-					Field field = entry.getValue();
-
-					return String.valueOf(field.getValue());
-				})
-		);
+		Map<String, Field> fields = document.getFields();
+
+		HashMapBuilder.HashMapWrapper<String, String> hashMapWrapper =
+			HashMapBuilder.create(fields.size());
```
Comment (dantewang) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/dto/v1_0/util/ProcessUtil.java:72]: @slnn Please don't use `HashMapBuilder` here. Just declare a `HashMap` named `titleMap` (matching the method name), and put key-values in the loop.

Additionally, we usually don't give `HashMap` or any other hash based data structure initial capacity, which is due to how hash data structures are implemented.

Please make changes to all the occasions in your PR and resend, thanks!

**Comment by slnn** [comment]
Update and resend in https://github.com/liferay-core-infra/liferay-portal/pull/4069

**Pull request closed without merging**